docs/testing.md

```diff
@@ -92,6 +92,32 @@ Deno.test({
 });
 ```
 
+### Exit sanitizer
+
+There's also the exit sanitizer which ensures that your code can't accidentally
+call Deno.exit by accident via your own code or third-party code signaling a
+false test success.
```

Comment:
The exit sanitizer which ensures that tested code doesn't
call `Deno.exit()` signaling a false test success.

Reply:
Yeah that does sound better, applied.